ContainerController/ContainerController.xcodeproj/project.pbxproj

```diff
@@ -250,6 +234,7 @@
 				ONLY_ACTIVE_ARCH = YES;
 				SDKROOT = iphoneos;
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
+				SWIFT_VERSION = 4.2;
```

Comment:
You also need to modify the podspec to contain 4.2 as `swift_version`. The Xcode project files are neither contained in Pods nor used to create the Pod